lib/sdk/request.js

```diff
@@ -165,6 +165,7 @@ const Response = Class({
     response(this).request = request;
   },
   get text() response(this).request.responseText,
+  get responseURL() response(this).request.responseURL,
```

Comment:
The other property names here appear to have removed the `response` prefix, so in order to follow that pattern this line should be:

    get url() response(this).request.responseURL